### PR TITLE
testing/gnome-terminal: new aport

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by maxice8** [comment]
gnome-terminal is in unmaintained/

File: testing/gnome-terminal/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributor: Rasmus Thomsen <oss@cogitri.dev>
+# Maintainer: Rasmus Thomsen <oss@cogitri.dev>
+pkgname=gnome-terminal
+pkgver=3.32.1
+pkgrel=0
+pkgdesc="GNOME terminal emulator application"
+url="https://wiki.gnome.org/Apps/Terminal"
+arch="all"
+license="GPL-2.0-or-later, GFDL-1.3-only"
```
Comment (maxice8) [inline testing/gnome-terminal/APKBUILD]: space without commad.

File: testing/gnome-terminal/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+depends="desktop-file-utils"
+makedepends="dconf-dev gnome-shell gsettings-desktop-schemas-dev nautilus-dev vte3-dev
+	glib-dev intltool itstool glib-dev"
+subpackages="$pkgname-doc $pkgname-lang"
+source="https://download.gnome.org/sources/gnome-terminal/${pkgver%.*}/gnome-terminal-${pkgver}.tar.xz
+	fix-W_EXITCODE.patch"
+builddir="$srcdir/$pkgname-$pkgver"
```
Comment (maxice8) [inline testing/gnome-terminal/APKBUILD]: default value

File: testing/tracker-miners/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	flac-dev gexiv2-dev gstreamer-dev enca-dev libcue-dev libexif-dev
+	libgsf-dev libgxps-dev libiptcdata-dev libjpeg-turbo-dev libosinfo-dev
+	libpng-dev libseccomp-dev tiff-dev libxml2-dev libvorbis-dev poppler-dev
+	taglib-dev totem-pl-parser-dev upower-dev zlib-dev gst-plugins-base-dev
+	giflib-dev"
+checkdepends="bash coreutils python py2-gobject3"
+options="!check" # Requires itself to be installed on the system
+subpackages="$pkgname-doc $pkgname-lang"
+source="https://download.gnome.org/sources/tracker-miners/${pkgver%.*}/tracker-miners-${pkgver}.tar.xz"
+builddir="$srcdir/tracker-miners-${pkgver}"
```
Comment (maxice8) [inline testing/tracker-miners/APKBUILD]: default value

File: testing/nautilus/APKBUILD [hunk 4]
```diff
@@ -1,45 +1,38 @@
...
-install=""
-subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-source="https://download.gnome.org/sources/nautilus/3.24/nautilus-$pkgver.tar.xz"
+makedepends="$depends_dev itstool libxml2-dev libxml2-utils docbook-xsl
+	docbook-xml libxslt intltool libexif-dev jpeg-dev tiff-dev librsvg-dev
+	libseccomp-dev tracker-dev gst-plugins-base-dev meson gexiv2-dev"
+checkdepends="tracker-miners"
+subpackages="$pkgname-dev $pkgname-lang"
+source="https://download.gnome.org/sources/nautilus/${pkgver%.*}/nautilus-$pkgver.tar.xz"
 builddir="$srcdir/nautilus-$pkgver"
```
Comment (maxice8) [inline testing/nautilus/APKBUILD]: default value

**Comment by Cogitri** [comment]
Needs #7379

**Comment by Cogitri** [comment]
Fixed, thanks.

**Pull request closed without merging**